src/objc_symbols_common.c

```diff
@@ -0,0 +1,104 @@
+
+/* Compiler implementation of the D programming language
+ * Copyright (c) 2015 by Digital Mars
+ * All Rights Reserved
+ * written by Michel Fortin
+ * http://www.digitalmars.com
+ * Distributed under the Boost Software License, Version 1.0.
+ * http://www.boost.org/LICENSE_1_0.txt
+ * https://github.com/D-Programming-Language/dmd/blob/master/src/objc_symbols_common.c
+ */
+
+#include "aggregate.h"
+#include "cc.h"
+#include "dt.h"
+#include "type.h"
+#include "objc.h"
+#include "global.h"
+
+// MARK: ObjcSymbols
+
+bool ObjcSymbols::hassymbols = 0;
+
+Symbol *ObjcSymbols::msgSend = NULL;
+Symbol *ObjcSymbols::msgSend_stret = NULL;
+Symbol *ObjcSymbols::msgSend_fpret = NULL;
+Symbol *ObjcSymbols::msgSend_fp2ret = NULL;
+
+Symbol *ObjcSymbols::siminfo = NULL;
+Symbol *ObjcSymbols::smodinfo = NULL;
+
+StringTable *ObjcSymbols::smethvarnametable = NULL;
+StringTable *ObjcSymbols::smethvarreftable = NULL;
+
+extern int seg_list[SEG_MAX];
+
+static StringTable *initStringTable(StringTable *stringtable)
+{
+    delete stringtable;
+    stringtable = new StringTable();
+    stringtable->_init();
+
+    return stringtable;
+}
```

Comment:
You can remove this function and call [`StringTable::reset(..)`][1] instead, which does the same thing.

[1]: https://github.com/D-Programming-Language/dmd/blob/master/src/root/stringtable.c#L135